base/cvd/cuttlefish/host/libs/web/BUILD.bazel

```diff
@@ -0,0 +1,54 @@
+cc_library(
+    name = "web",
+    srcs = [
+        "android_build_api.cpp",
+        "android_build_string.cpp",
+        "caching_build_api.cpp",
+        "chrome_os_build_string.cpp",
+        "credential_source.cc",
+        "http_client/http_client.cc",
+        "http_client/http_client_util.cc",
+        "luci_build_api.cpp",
+    ],
+    hdrs = [
+        "android_build_api.h",
+        "android_build_string.h",
+        "caching_build_api.h",
+        "chrome_os_build_string.h",
+        "credential_source.h",
+        "http_client/http_client.h",
+        "http_client/http_client_util.h",
+        "luci_build_api.h",
+    ],
+    copts = [
+        "-std=c++17",
+    ],
+    linkopts = ["-lcurl"],
```

Comment:
This target (any maybe ones that depend on it) should be the only one that needs `lcurl`.

Reply:
Done